### ISSUE TITLE
Wordpress database error on screen search

### ISSUE THREAD

**Issue by teledirigido** [juliantroeps/search-everything-fork#5]
I found an error when I click on the admin screen search button and the query is empty:

WordPress database error: [You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '))) AND wp_posts.post_type = 'properties' AND (wp_posts.post_status = 'publish'' at line 1]
```
SELECT SQL_CALC_FOUND_ROWS wp_posts.ID FROM wp_posts WHERE 1=1 AND ((())) AND wp_posts.post_type = 'properties' AND (wp_posts.post_status = 'publish' OR wp_posts.post_status = 'acf-disabled' OR wp_posts.post_status = 'future' OR wp_posts.post_status = 'draft' OR wp_posts.post_status = 'pending' OR wp_posts.post_status = 'private') ORDER BY wp_posts.post_date DESC LIMIT 0, 20
```

**Comment by juliantroeps** [comment]
Thanks! I'll look into that.

**Comment by aurora** [comment]
I had a similar issue, but i think this should be fixed by #4 , no?

**Comment by juliantroeps** [comment]
@aurora Yep, seems to fixed that.
